<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeDialog</class>
 <widget class="QDialog" name="UpgradeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gridcoin Upgrading Facility</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/bkg);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="pixmap">
      <pixmap resource="../bitcoin.qrc">:/images/about</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="retryDownloadButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Retry Download</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upgradeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Upgrade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hideButton">
         <property name="text">
          <string>Hide</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
